guides/release/components/built-in-components.md

```diff
@@ -114,28 +113,15 @@ Starting with Ember Octane, we recommend using the `{{on}}` modifier to call an
 ```handlebars
 <label for="input-name">Name:</label>
 <Input
-  @id="input-name"
+  id="input-name"
   @value={{this.name}}
   {{on "input" this.validateName}}
 />
 ```
 
-The event name (e.g. `"focusout"`, `"input"`, `"keydown"`) always follows the casing that the HTML standard uses.
+[Learn more about the `{{on}}` modifier.](../../upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier)
 
-For backwards compatibility with earlier versions of Ember, it is possible to call an action by passing an event argument.
-
-```handlebars
-<label for="input-name">Name:</label>
-<Input
-  @id="input-name"
-  @value={{this.name}}
-  @input={{this.validateName}}
```

Comment:
removing this example with  backwards compatibility, because this usage gonna be deprecated per RFC707